Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -288,3 +291,21 @@ public struct WorldScaleGeoTrackingSceneView: View {
         }
     }
 }
+
+extension WorldScaleGeoTrackingSceneView {
+    @MainActor
+    class ViewModel: ObservableObject {
```

Comment:
It's unclear if this view model is for the calibration view or the WorldScaleGeoTrackingSceneView. 

I think that's because it's used by both. To me it seems like the CalibrationView doesn't need this view model at all, it simply needs bindings to heading and elevation. From there the WorldScaleGeoTrackingSceneView can pass bindings to state to it, and the calibration view can update the state. Then the WorldScaleGeoTrackingSceneView can implement onChange for those values and update the camera controller. So I'm not sure this view model is needed.

Reply:
Removed the view model as it was not needed.